src/System.Diagnostics.FileVersionInfo/tests/Assembly1.csproj

```diff
@@ -30,7 +30,7 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="System.Runtime">
-      <HintPath>..\..\packages\System.Runtime.4.0.20-beta-22412\lib\portable-wpa80+win80+net45+aspnetcore50\System.Runtime.dll</HintPath>
+      <HintPath>..\..\..\packages\System.Runtime.4.0.20-beta-22412\lib\portable-wpa80+win80+net45+aspnetcore50\System.Runtime.dll</HintPath>
```

Comment:
If I remember correctly this project isn't building like the other projects so we may need to keep this reference (I'm happy to be wrong here). If we do keep it then please do update it to use $(PackagesDir) like @krwq suggests.